src/index.js

```diff
@@ -1,11 +1,42 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
 import './index.css';
+import {
+  BrowserRouter as Router, Routes,
+  Route,
+} from 'react-router-dom';
 import App from './App';
+import Navbar from './components/Navbar';
+import Home from './components/Home';
+import Quote from './components/Quote';
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>,
+  <Router>
+    <Navbar />
+    <Routes>
+      <Route path="/" element={<Home />} />
+      <Route path="/app" element={<App />} />
+      <Route path="/quote" element={<Quote />} />
+    </Routes>
+  </Router>,
+  /* <Router>
+    <div>
+      <Navbar />
+      <div className="content">
+        <Switch>
+          <Route exact path="/">
+            <Home />
+          </Route>
+          <Route path="/app">
+            <App />
+          </Route>
+          <Route path="/quote">
+            <Quote />
+          </Route>
+        </Switch>
+      </div>
+    </div>
+  </Router>, */
```

Comment:
- [ ] Kindly remove the commented code, its not good practice to have commented code in your codebase.